com.unity.render-pipelines.universal/Editor/Camera/UniversalRenderPipelineCameraUI.Drawers.cs

```diff
@@ -0,0 +1,92 @@
+
+        public static readonly CED.IDrawer SectionProjectionSettings = CED.FoldoutGroup(
+            CameraUI.Styles.projectionSettingsHeaderContent,
+            CameraUI.Expandable.Projection,
+            k_ExpandedState,
+            FoldoutOption.Indent,
+            CED.Group(
+                Drawer_Projection
+                ),
+            PhysicalCamera.Drawer,
+            CED.Group(
+                CameraUI.Drawer_FieldClippingPlanes
+            )
+        );
+
+        public static readonly CED.IDrawer SectionStackSettings =
+            CED.Conditional(
+                (serialized, editor) => (CameraRenderType)serialized.cameraType.intValue == CameraRenderType.Base,
+                CED.FoldoutGroup(Styles.stackSettingsText, CameraUI.Expandable.Stack, k_ExpandedState, FoldoutOption.Indent, CED.Group(Drawer_StackCameras)));
+
+        public static readonly CED.IDrawer[] Inspector =
+        {
+            CED.Group(
+                Drawer_CameraType
+                ),
+            SectionProjectionSettings,
+            Rendering.Drawer,
+            SectionStackSettings,
+            Environment.Drawer,
+            Output.Drawer
+        };
+
+        static void Drawer_Projection(UniversalRenderPipelineSerializedCamera p, Editor owner)
+        {
+            var camera = p.serializedObject.targetObject as Camera;
+
+            camera.TryGetComponent<UnityEngine.Experimental.Rendering.Universal.PixelPerfectCamera>(out var pixelPerfectCamera);
+
+            bool pixelPerfectEnabled = pixelPerfectCamera != null && pixelPerfectCamera.enabled;
+            if (pixelPerfectEnabled)
+                EditorGUILayout.HelpBox(Styles.pixelPerfectInfo, MessageType.Info);
+
+            using (new EditorGUI.DisabledGroupScope(pixelPerfectEnabled))
+                CameraUI.Drawer_Projection(p, owner);
+        }
+
+        static void Drawer_CameraType(UniversalRenderPipelineSerializedCamera p, Editor owner)
+        {
+            int selectedRenderer = p.renderer.intValue;
+            ScriptableRenderer scriptableRenderer = UniversalRenderPipeline.asset.GetRenderer(selectedRenderer);
+            bool isDeferred = scriptableRenderer is UniversalRenderer renderer ? renderer.renderingMode == RenderingMode.Deferred : false;
+
+            EditorGUI.BeginChangeCheck();
+
+            CameraRenderType originalCamType = (CameraRenderType)p.cameraType.intValue;
+            CameraRenderType camType = (originalCamType != CameraRenderType.Base && isDeferred) ? CameraRenderType.Base : originalCamType;
+
+            camType = (CameraRenderType)EditorGUILayout.EnumPopup(
+                Styles.cameraType,
+                camType,
+                e =>
+                {
+                    return isDeferred ? (CameraRenderType)e != CameraRenderType.Overlay : true;
+                },
+                false
+            );
```

Comment:
I think we should add space here. As I feel like RenderType property is too close to projection group
![image](https://user-images.githubusercontent.com/24258017/116992713-21d19580-acdf-11eb-8de0-3ed8872b1891.png)
This also makes borders very close
@TheoWong-pixel WDYT?

Reply:
AFAIK thats standard spacing (@alex-vazquez please confirm) and additional spaces are not really a thing we are supposed to do. The closeness is more related to the sub-header design rather than this one UI. So i would rather we do a separate PR to change that for all sub-headers rather than custom change this one.